src/skmultiflow/trees/nodes/ada_split_node_for_regression.py

```diff
@@ -187,3 +186,17 @@ def filter_instance_to_leaves(self, X, y, weight, parent, parent_branch,
         if self._alternate_tree is not None:
             self._alternate_tree.filter_instance_to_leaves(X, y, weight, self, -999,
                                                            update_splitter_counts, found_nodes)
+
+    def get_normalized_error(self, y, y_pred):
```

Comment:
Should we consider making this a generic utility function?

Reply:
We could easily do that. But in this case, an instance of the learning node should be passed to the function to get access to the inner statistics (maximum and minimum observed errors). We would also have to state in the method's docstring that the passed learning node instance must have the `_min_error` and `_max_error` fields. What do you think?